addons/binding/org.openhab.binding.nest/ESH-INF/thing/camera.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="nest" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="camera" listed="false">
```

Comment:
I wonder a bit, what the use case of supporting the camera exactly is. The "url" channels are not useful to the user, i.e. they cannot be used in a UI. The others are mainly for configuring the camera - is this really something one wants to do through openHAB?